toolchains/osx.toolchain.cmake

```diff
@@ -39,17 +39,40 @@ set( TNUN_os_suffix OSX )
 
 set( TNUN_cpu_archs default )
 
+set( TNUN_ABI x64 )
```

Comment:
In MB Source when we give names to tests, we add ABI suffix to be able to distinguish them in CI server report. In old build system this was PHOTOPAY_ARCHITECTURE variable which is now TNUN_ABI. On MacOS we always have x64 ABI.

Generally, this can be removed from build, but that would then require us to have parallel osx.toolchain.cmake file that would have to be loaded after this one.